Cargo.lock

```diff
@@ -448,7 +448,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "sundial"
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
not sure why this got pulled in...

should i add this in a separate PR?

Reply:
Yes please